server/eclipse-project/src/main/java/us/freeandfair/corla/model/NEBAssertion.java

```diff
@@ -6,11 +6,11 @@
 package us.freeandfair.corla.model;
 
 
+import au.org.democracydevelopers.raire.assertions.AssertionAndDifficulty;
```

Comment:
What's the reason for the addition of these imports? I can't see where they are used.